woocommerce/woocommerce-gutenberg-products-block
# Store API: Add cart coupon endpoints

Adds endpoints to add, remove, and get coupons from the Store API.

Closes #1323 

Docs are updated also. _Note_ Whilst working on that I noticed we didn't update the price formats for products and filters - that will be a follow up task along with more updated docs.

### How to test the changes in this Pull Request:

1. Add items to your cart, create 2 coupons, and add 1 to your cart manually. This is so you can test the responses. 
2. Check `wc/store/cart` lists applied coupons.
3. Check `wc/store/cart/coupons` lists applied coupons.
4. For one of the listed coupons, try and access it's endpoint: `wc/store/cart/coupons/<code>`
5. For one of the listed coupons, try and delete it DELETE request to `wc/store/cart/coupons/<code>`
6. Add it again, POST `{ "code": "<code>" }`  to `wc/store/cart/coupons`
7. Finally, delete all coupons by doing DELETE request to `wc/store/cart/coupons`
